app/controllers/idv/address_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Idv
+  class AddressController < ApplicationController
+    include IdvSession
+    include IdvFailureConcern
+
+    attr_reader :idv_form
+
+    before_action :confirm_two_factor_authenticated
+
+    def new
+      analytics.track_event(Analytics::IDV_ADDRESS_VISIT)
+      set_idv_form
+      render 'idv/address'
```

Comment:
If you move `idv/address.html.slim` to `idv/address/new.html.slim` you can get rid of this `render` call.